openwisp_monitoring/device/apps.py

```diff
@@ -134,6 +140,13 @@ def device_post_delete_receiver(cls, instance, **kwargs):
         instance.checks.all().delete()
         instance.metrics.all().delete()
 
+    @classmethod
+    def organization_post_save_receiver(cls, instance, *args, **kwargs):
+        if instance.is_active is False:
+            from .tasks import organization_disabled_handler
```

Comment:
is the import here necessary?

Reply:
Yes, because `openwisp_monitoring.device.tasks` imports `openwisp_monitoring.checks.tasks.perform_check` which in-turn imports the `ContentType` model globally.